### ISSUE TITLE
[Bug]: no aria-labeling for the facebook, instagram and webpages on the school pages

### ISSUE THREAD

**Issue by BeeSeeWhy** [sfbrigade/support-sfusd#422]
### What happened?
The individual school pages template does not have aria-labelling for the three links at the top: Facebook, Instagram and school website. 

<img width="856" height="215" alt="Image" src="https://github.com/user-attachments/assets/de8e5bae-5121-4f7e-826b-bc3ddfd730a6" />

<img width="1363" height="418" alt="Image" src="https://github.com/user-attachments/assets/74aee19f-da45-492b-b3ae-8834da57a020" />

### What should have happened instead?

We need to add to the school page template an aria label

### Version

supportsfschools.org (Production)

### What browser(s) are you seeing the problem on?

Chrome 

### Which platform(s) are you seeing the problem on?

MacOS
